habitat-baselines/habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -148,11 +148,13 @@ def __init__(
     ):
         super().__init__()
         self.no_downscaling = no_downscaling
+        # Goal/privileged information sensors
+        skip_sensors = {ImageGoalSensor.cls_uuid, "robot_head_panoptic"}
```

Comment:
Would really prefer to use the UUID here

Reply:
The uuid is in [default_structured_configs](https://github.com/facebookresearch/habitat-lab/blob/df89d7c3c951ae02aeb0bb57c6ac975bb789247c/habitat-lab/habitat/config/default_structured_configs.py#L1064). Importing that here does not seem right. 

But @JHurricane96, is excluding this from gym.obs_keys not sufficient?

Reply:
It looks like it is sufficient. We can remove this from here.

Reply:
Thanks for checking. Will remove